js/mathOperations.js

```diff
@@ -77,28 +80,76 @@ function create()
   myscore = game.add.text(80, 43 , '000' , {font : "15px Arial" , fill : "#eceff1"});
   mylevel = game.add.text(311, 43 , '01' , {font : "15px Arial" , fill : "#eceff1"});
   timer = game.add.text(515, 43, '00:00:00' ,{font : "15px Arial" , fill : "#eceff1"});
-
-  plus = game.add.sprite(150 , 350 , 'plus');
-  minus = game.add.sprite(250 , 350 , 'minus');
-  multiply = game.add.sprite(350 , 350 , 'multiply');
-  divide = game.add.sprite(450 , 350 , 'divide');
-
+  //if(level < 2)
+  //{
+   //plus = game.add.sprite(150 , 350 , 'plus');
+   //minus = game.add.sprite(250 , 350 , 'minus');
+   //multiply = game.add.sprite(350 , 350 , 'multiply');
+   //divide = game.add.sprite(450 , 350 , 'divide');
+   //power = game.add.sprite( 1000 , 1000 , 'power')
+  //} 
+  //else
+  //{ 
+  plus = game.add.sprite(50 , 350 , 'plus');
```

Comment:
Why are you changing these values?  If it has to do with where something gets put on the screen, these values should be calculated based on the screen dimensions shouldn't they?  Not saying your change is wrong, just looking for a justification.